examples/speed_estimation/setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Get the directory where the script is located
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+# Check if 'data' directory does not exist and then create it
+if [[ ! -e $DIR/data ]]; then
+    mkdir "$DIR/data"
+else
+    echo "'data' directory already exists."
+fi
+
+# Download the vehicles.mp4 file from Google Drive
+gdown -O "$DIR/data/vehicles.mp4" "https://drive.google.com/uc?id=1pz68D1Gsx80MoPg-_q-IbEdESEmyVLm-"
```

Comment:
Suggestion: We have our asset system so we don't need use gdown anymore or script like this we can download via python script (operation system independent)